docs/user_guides/fs/storage_connector/creation/jdbc.md

```diff
@@ -36,16 +36,26 @@ Head to the Storage Connector View on Hopsworks (1) and set up a new storage con
 
 Enter the details for your JDBC enabled database.
 
+<figure markdown>
+  ![JDBC Connector Creation](../../../../assets/images/guides/fs/storage_connector/jdbc_creation.png)
+  <figcaption>JDBC Connector Creation Form</figcaption>
+</figure>
+
 1. Select "JDBC" as connector protocol.
 2. Enter the JDBC connection url. This can for example also contain the username and password.
-3. Add additional key/value arguments to be passed to the connection. These might differ by database. It can be the username and password.
+6. Add driver class name as an argument. This is a mandatory argument even if using default MySQL driver. Add an argument with name `driver` and driver class name as value. For MySQL databases, the class name is `com.mysql.cj.jdbc.Driver`, as shown in example below. The driver class name will differ based on the database to connect.
```

Comment:
i dont think this step should be nr. 6

Reply:
Maybe it should be as a sub-step of nr. 3? since it is about arguments. Or it could be a note.

Reply:
changed it to a  note